content/api/servo-assembly/_index.md

```diff
@@ -8,6 +8,10 @@ The Servo Assembly is used to connect to multiple types of peripheral:
 - [Servos](servos)
 - [Ultrasound sensors](ultrasound)
 
+At the core of the Servo Assembly is an Arduino. If you need more direct control
```

Comment:
Not a big deal, but is there any reason why "Arduino" here isn't hyperlinked in the same way that the occurrences in the introductions to `arduino-commands.md` and `kit/servo-assembly/_index.md` are?

Reply:
Only that it didn't seem entirely useful to do so. Anyone wanting more detail will probably be on one of those pages anyway. Do we prefer to be over-linking?

Reply:
Fair enough, good point.